src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.targets

```diff
@@ -7,8 +7,4 @@
       Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' "
       Project="Sdk.targets" Sdk="Microsoft.Maui.Controls.Sdk"
   />
-  <Import
-      Condition=" '$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true' "
-      Project="Sdk.targets" Sdk="Microsoft.Maui.Resizetizer.Sdk"
-  />
```

Comment:
@jonathanpeppers I see this was removed so no `$(UseMauiAssets)` no longer applies. However, since this is now a dependency of the `Microsoft.Maui.Controls.Sdk`, we might also need to add the condition to the line above?

It doesn't seem to pull in anything else.

Reply:
Good catch, it should be:

```xml
  <Import
      Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true' "
      Project="Sdk.targets" Sdk="Microsoft.Maui.Controls.Sdk"
  />
```

One confusing thing is the name of this SDK, it has the name "Controls", but it's imported event when you're not using Controls.

Should we rename this one to Microsoft.Maui.Sdk? I would probably do that in a future PR.

Reply:
Yeah, we can do that in a new PR so we can merge this one now.